client/my-sites/site-settings/podcasting-details/index.jsx

```diff
@@ -257,7 +258,24 @@ class PodcastingDetails extends Component {
 	}
 
 	onCategorySelected = category => {
-		this.setFieldForcingString( 'podcasting_category_id' )( category.ID );
```

Comment:
There only seems to be one remaining use of this method now -- perhaps we should remove it.

Reply:
Good point - removed.